illumos-utils/src/zone.rs

```diff
@@ -17,6 +17,7 @@ use crate::dladm::{EtherstubVnic, VNIC_PREFIX_BOOTSTRAP, VNIC_PREFIX_CONTROL};
 use crate::{execute, PFEXEC};
 use omicron_common::address::SLED_PREFIX;
 
+pub const CHRONYD: &str = "/usr/sbin/chronyd";
```

Comment:
I don't think you need this line any more.

Reply:
Ha! no, I'll update